app/models/version.rb

```diff
@@ -21,13 +21,16 @@ def self.tags
 
   # Find or create the default version
   def self.default
+    return local if Rails.env.development?
     find_by(default: true) || create(default: true, tag: "latest")
   end
 
   # Find or create the local version
   def self.local
     if Rails.env.development?
-      find_by(tag: "local") || create!(tag: "local", url_prefix: 'local')
+      version = find_by(tag: "local") || create!(tag: "local")
+      version.update!(default: true) unless version.default?
+      version
```

Comment:
Let's think about this some more: how should this structure act in the basics? What is a local version, and what is a default version: are they the same or not? What is their difference in how they should act?

The setup we have now looks too complicated and with too may references back and forth.